website/docs/installation/weave-gitops-enterprise/index.mdx

```diff
@@ -9,21 +9,15 @@ import TabItem from "@theme/TabItem";
 import TierLabel from "../../_components/TierLabel";
 import CurlCodeBlock from "../../_components/CurlCodeBlock";
 
-## Installing Weave GitOps Enterprise<TierLabel tiers="enterprise" />
+## Installing Weave GitOps Enterprise<TierLabel tiers="Enterprise" />
 
-Weave GitOps Enterprise (WGE) provides ops teams with an easy way to assess the
-health of multiple clusters in a single place. It shows cluster information such as
-Kubernetes version and number of nodes and provides details about the GitOps operations
-on those clusters, such as Git repositories and recent commits. Additionally, it
-aggregates Prometheus alerts to assist with troubleshooting.
+:::info
+To purchase entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
 
-To purchase entitlement to Weave GitOps Enterprise please contact [sales@weave.works](mailto:sales@weave.works)
-
-:::note 
-There is no need to install Weave GitOps (OSS) before installing Weave GitOps Enterprise
+For more information about WGEE, see the [Enterprise feature page](../../intro-ee.mdx).
```

Comment:
Is the acronym WGEE clear enough?

Reply:
If not I can put the full version. The only reason I had the acronym here is because it was written in full just above, but no harm in doing it again if that would be clearer

Reply:
I'd prefer to not abbreviate it as we don't usually do that in the docs.